content/2025/09-05-metadata-view-enhancements.md

```diff
@@ -0,0 +1,54 @@
+---
+applied_at: '2025-09-05'
+applies_to:
+- ui-elements
+is_impactful: false
+is_new_feature: false
+release_source_url: ''
+collapse: true
+show_excerpt: true
+---
+
+# Metadata view enhancements
+
+We introduced the following updates to the [metadata view in Content Explorer][1]:
+
+## For developers 
+
+- Enable or disable specific metadata columns and arrange their order. You can also
```

Comment:
Can we double check this list of features? There is some misalignment on what's avaliable